src/content/docs/pl/concepts/why-astro.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Dlaczego Astro?
+description: "Astro to wszechstronny framework do budowania szybkich stron internetowych skoncentrowanych na dostarczaniu treści. Dowiedz się więcej."
+i18nReady: true
+---
+
+Astro to **wszechstronny** **framework** do budowania **szybkich** stron internetowych skoncentrowanych na **dostarczaniu treści**.
+
+Dlaczego wybrać Astro, a nie inny webowy framework? Oto pięć głównych projektowych zasad, które pomogą wytłumaczyć dlaczego zbudowaliśmy Astro, problemy, które rozwiązuje, i dlaczego Astro może być najlepszym wyborem dla Twojego projektu lub zespołu:
+
+#### Astro jest...
+
+1. [Skoncentrowany na treści](#skoncentrowany-na-treści): Astro jest zaprojektowane dla stron bogatych w treść.
+2. [Server-first](#server-first): Strony działają szybciej, kiedy HTML jest generowany na serwerze.
+3. [Domyślnie szybki](#domyślnie-szybki): Zbudowanie wolnej strony za pomocą Astro powinno być niemożliwe.
+4. [Łatwy w użyciu](#łatwy-w-użyciu): Nie musisz być ekspertem, żeby budować z Astro.
+5. [Pełny funkcji, ale elastyczny](#pełny-funkcji-ale-elastyczny): Ponad 100 integracji z Astro do wyboru.
+
+## Skoncentrowany na treści
+
+**Astro zostało zaprojektowane do budowania stron bogatych w treść.** Należą do nich większość stron marketingowych, wydawniczych, dokumentacje, blogi, portfolia oraz niektóre strony e-commerce.
+
+W przeciwieństwie do tego podejścia większość współczesnych webowych frameworków jest zaprojektowana w celu budowania *aplikacji internetowych*. Te frameworki sprawdzają się najlepiej przy tworzeniu bardziej zaawansowanych, przypominających aplikacje, doświadczeń: panele admina, skrzynki odbiorcze, serwisy społecznościowe, listy zadań, a nawet aplikacje podobne do natywnych jak [Figma](https://figma.com/) lub [Ping](https://ping.gg/).
```

Comment:
```suggestion
W przeciwieństwie do tego podejścia większość współczesnych webowych frameworków jest zaprojektowana w celu budowania *aplikacji internetowych*. Te frameworki sprawdzają się najlepiej przy tworzeniu bardziej zaawansowanych, przypominających aplikacje na przykład: panele admina, skrzynki odbiorcze, serwisy społecznościowe, listy zadań, a nawet aplikacje podobne do natywnych jak [Figma](https://figma.com/) lub [Ping](https://ping.gg/).
```

Reply:
zgadzam się, że brakuje tego "na przykład", ale nie można całkiem wyrzucić "doświadczeń", bo wtedy "zaawansowanych, przypominających aplikacje" nie odnosi się do niczego. Zamieniłem to na "stron", żeby było bardziej czytelne